cvmfs/talk.cc

```diff
@@ -510,11 +511,12 @@ static void *MainTalk(void *data __attribute__((unused))) {
 /**
  * Init the socket.
  */
-bool Init(const string &cachedir) {
+bool Init(const string &cachedir, OptionsManager *optionsManager) {
```

Comment:
I forgot to change the variable name to something like options_manager, according to the Google C++ style guide.